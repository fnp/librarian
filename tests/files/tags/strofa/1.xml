<autor_utworu>Mikołaj Sęp Szarzyński</autor_utworu>
<nazwa_utworu>Do Zosie</nazwa_utworu>

<strofa>Będę się zawsze dziwował twojej pikności:/
Nie szkodzi wiernej dalekość miłości;/
Bo, gdzie ciałem nie mogę być, tam myślą będę,/
A pierwej siebie, niż ciebie zabędę;/
A dotąd się serce me smęcić nie przestanie,/
Aż cię oglądam, me wdzięczne kochanie.
</strofa>
