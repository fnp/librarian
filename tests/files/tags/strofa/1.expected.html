<div id="book-text">
<h1>
<span class="wl author" id="i1">Mikołaj Sęp Szarzyński</span><span class="wl title" id="i2">Do Zosie</span>
</h1>
<div class="stanza" id="i3">
<a href="#f1" class="wl-num">1</a><div class="wl verse" id="f1">Będę się zawsze dziwował twojej pikności:</div>
<div class="wl verse" id="f2">Nie szkodzi wiernej dalekość miłości;</div>
<div class="wl verse" id="f3">Bo, gdzie ciałem nie mogę być, tam myślą będę,</div>
<div class="wl verse" id="f4">A pierwej siebie, niż ciebie zabędę;</div>
<a href="#f5" class="wl-num">5</a><div class="wl verse" id="f5">A dotąd się serce me smęcić nie przestanie,</div>
<div class="wl verse" id="f6">Aż cię oglądam, me wdzięczne kochanie.</div>
</div>
</div>
